### DISCUSSION TITLE
ebusd global updatecheck in home assistant : how to update

### DISCUSSION THREAD

**Discussion by jjtoard** [john30/ebusd#1354] [Q&A]
ebusd global updatecheck in home assistant : how to update

the entity "update.ebusd_global_updatecheck" reports that an update is available but I don't see options to update it.
How can I update this ?  The device is on the latest update.

![afbeelding](https://github.com/user-attachments/assets/0fd8b823-6b62-4459-8b6a-595e7eb20e2e)

**Comment by InToSSH** [comment]
@jjtoard I don't think this has anything to do with ebusd repo directly. This issue tells you that there is a new version of ebusd, however since the actual HA addon that wraps the ebusd service is in a different repo (https://github.com/LukasGrebe/ha-addons/tree/main/ebusd) it needs to be updated to mirror the latest ebusd version.

**Comment by janchlebek** [comment]
Hi, the same thing happened to me now - what do you mean by "it needs to be updated to mirror the latest ebusd version"?
